schutzbot/deploy.sh

```diff
@@ -52,6 +52,7 @@ ARCH=$(uname -m)
 
 if [[ $ID == "rhel" && $VERSION_ID == "8.3" && -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
     greenprint "Registering RHEL"
+    sudo subscription-manager remove --all
```

Comment:
Is this going to remove all subscriptions on the account? Including ones from other systems? 

If yes this is prone to race conditions with other test jobs running in parallel which subscribe and then we remove their subscriptions before these systems were able to execute their dnf commands. That would lead to even more peculiar and hard to debug cases if it is really the case. 

Is there a way to remove all previous subscriptions only from the same system ?

Reply:
I've read a bit about how subscription manager works and I think it only removes the subscription from this system ([Removing a subscription](https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html-single/rhsm/index#reg-cli-remove))

But I could be getting it wrong. Maybe just doing retry logic is faster (but not as clean imo)